bin/react-to-product-owners-yml-changes.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import os, sys, yaml
+from os.path import realpath, join, dirname
+
+# Run from project root.
+os.chdir(realpath(join(dirname(sys.argv[0]), '..')))
+
+PREFIX = 'Product Area: '
+LABELS_YAML = '.github/labels.yml'
+
+product_owners = yaml.safe_load(open(sys.argv[1]))
+labels = open(LABELS_YAML)
+
+fastforward = False
+head = []
+product_areas = ["- name: 'Product Area: Unknown'\n", "  color: '8D5494'\n"]
+tail = []
+current = head
+
+# Best to look the other way, Buck. This is just waaaay easier than trying to
+# use ruamel.yaml to preserve comments and other formatting.
+
+for line in labels:
```

Comment:
Took a while to understand what's going on here, can we include some comments for why `fastforward` has to be used? Also describing head as the parts of the file before the `Product Area` labels and tail as the parts of the file after the `Product Area` labels

Reply:
a4f16c85aa2227e72b0e160a7d48af95ed8fb8c6